lib/definitions/time.js

```diff
@@ -2,7 +2,21 @@ var time;
 var daysInYear = 365.25;
 
 time = {
-  ms: {
+  ns: {
+    name: {
+      singular: 'Nanosecond'
+    , plural: 'Nanoseconds'
+    }
+  , to_anchor: 1/1000000000
+  }
+,  mus: {
```

Comment:
In theory we could be using the offical `μs`, unsure if it's worth the thrill though?